x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/custom_assets_extension.tsx

```diff
@@ -8,13 +8,30 @@
 import React from 'react';
 import { type CustomAssetsAccordionProps, CustomAssetsAccordion } from '@kbn/fleet-plugin/public';
 import { i18n } from '@kbn/i18n';
+import { useLocation } from 'react-router-dom';
+import { EuiSpacer } from '@elastic/eui';
 import { useKibana } from '../../common/hooks/use_kibana';
 import { benchmarksNavigation, cloudPosturePages } from '../../common/navigation/constants';
 
 const SECURITY_APP_NAME = 'securitySolutionUI';
 
 export const CspCustomAssetsExtension = () => {
   const { application } = useKibana().services;
+  const { search } = useLocation();
+  const isCNVM = search.includes('vuln_mgmt');
```

Comment:
let's use `vuln_mgmt` from the constants.ts file, in case the integration name changes in the future